core/controlplane/config/templates/cloud-config-controller

```diff
@@ -345,11 +347,16 @@ coreos:
       command: start
       content: |
         [Unit]
-        Wants=kubelet.service docker.service
-        After=kubelet.service
+        Wants=kubelet.service docker.service install-kube-system.service
+        After=kubelet.service install-kube-system.service
 
         [Service]
-        Type=oneshot
+        Type=simple
+        Restart=on-failure
+        RestartSec=60
```

Comment:
I'm wondering why we don't need `RemainAfterExit=true` for this even though we have one for `install-kube-system.service`.
Perhaps because there are no units checking activeness of cfn-signal by running `systemctl is-active cfn-signal`, right?